Src/Fido2/Objects/DevicePublicKeyAuthenticatorOutput.cs

```diff
@@ -1,86 +1,85 @@
 ﻿#nullable disable
+namespace Fido2NetLib.Objects;
 
 using System;
 using Fido2NetLib.Cbor;
 
-namespace Fido2NetLib.Objects
+
+public sealed class DevicePublicKeyAuthenticatorOutput
 {
-    public sealed class DevicePublicKeyAuthenticatorOutput
+    // https://w3c.github.io/webauthn/#sctn-device-publickey-attestation-calculations
+    internal readonly byte[] _dpkAuthDataPrefix = Convert.FromHexString("64657669636520626f756e64206b6579206174746573746174696f6e2073696700ffffffff");
+
+    internal CborMap _attObj;
+
+    public byte[] AuthData => DataHelper.Concat(_dpkAuthDataPrefix, AaGuid.ToByteArray());
+
+    public byte[] Hash => DataHelper.Concat(DevicePublicKey.GetBytes(), Nonce);
+
+    public byte[] GetBytes() => _attObj.Encode();
+
+    public byte[] AuthenticationMatcher => DataHelper.Concat(AaGuid.ToByteArray(), DevicePublicKey.GetBytes());
```

Comment:
Do we need to consider the Endianness of the AaGuid when building the Authentication Matcher?

Reply:
Not as long as we're consistent with the storage during registration vs logon, which we will be.